src/test/signalk/SKNMEA2000ParserTest.cpp

```diff
@@ -83,5 +83,35 @@ TEST_CASE("SKNMEA2000Parser: Basic tests") {
     CHECK( update.getEnvironmentDepthBelowKeel() == 40.8 );
     CHECK( update.getEnvironmentDepthTransducerToKeel() == 1.2 );
   }
-}
 
+  SECTION("127250: VESSEL True HEADING RAPID ") {
+    SetN2kTrueHeading(msg, 0, 180);
+    const SKUpdate &update = p.parse(SKSourceInputNMEA2000, msg, SKTime(0));
+    CHECK( update.getSize() == 1);
+    CHECK( update.getNavigationHeadingTrue() == 180 );
+  }
+
+  SECTION("127250: VESSEL Magnetic HEADING RAPID ") {
+    SetN2kMagneticHeading(msg, 0, 181, -2, 3);
+    const SKUpdate &update = p.parse(SKSourceInputNMEA2000, msg, SKTime(0));
+    CHECK( update.getSize() == 2);
+    CHECK( update.getNavigationHeadingMagnetic() == 181 );
+    // CHECK( update.getNavigationMagneticDeviation() == -2 );
+    CHECK( update.getNavigationMagneticVariation() == 3 );
+  }
+
+  SECTION("127245: Rudder Angle") {
+    SetN2kRudder(msg,-3.5, 0);
+    const SKUpdate &update = p.parse(SKSourceInputNMEA2000, msg, SKTime(0));
+    CHECK( update.getSize() == 1);
```

Comment:
For some reason this test failed too: https://travis-ci.org/sarfata/kbox-firmware/builds/308718735#L1716

Apparently the message was not processed at all (test output says `update.getSize() == 0`).

Reply:
now I (hope to) know more about this tests I will do them before doing a PR
In this case it could be the angle too (which should be rad again)